include/eve/concept/memory.hpp

```diff
@@ -27,7 +27,7 @@ namespace eve
 
   template<typename T, typename SIMD>
   concept simd_compatible_ptr = simd_value<SIMD> && (   std::is_pointer_v<T>
-                                                    ||  detail::is_aligned_ptr<T>::value
-                                                    && (pointer_alignment_v<T> >= alignment_v<SIMD>)
+                                                    ||  (detail::is_aligned_ptr<T>::value
+                                                         && (pointer_alignment_v<T> >= alignment_v<SIMD>))
```

Comment:
|| and && mixing without brackets warning

Reply:
I got this warning once but never had to pop it back.